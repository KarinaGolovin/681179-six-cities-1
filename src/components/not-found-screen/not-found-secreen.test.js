import React from 'react';
import NotFoundScreen from './not-found-screen.jsx';
import ShallowRenderer from 'react-test-renderer/shallow';

const renderer = new ShallowRenderer();

it(`NotFoundScreen correctly renders`, () => {
  const component = renderer.render(
      <NotFoundScreen />
  );

  expect(component).toMatchSnapshot();
});

