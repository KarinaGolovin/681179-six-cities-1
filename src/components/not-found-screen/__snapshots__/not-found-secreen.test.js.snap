// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFoundScreen correctly renders 1`] = `
<Memo(ConnectFunction)>
  <h1>
    Page Not Found
  </h1>
</Memo(ConnectFunction)>
`;
